scripts/globals/mobskills.lua

```diff
@@ -557,14 +557,20 @@ xi.mobskills.mobFinalAdjustments = function(dmg, mob, skill, target, attackType,
         target:delStatusEffect(xi.effect.THIRD_EYE)
     end
 
-    if attackType == xi.attackType.PHYSICAL and not skill:isSingle() then
-        target:delStatusEffect(xi.effect.THIRD_EYE)
-    end
+    if
+        attackType == xi.attackType.PHYSICAL or
+        attackType == xi.attackType.RANGED
+    then
+        if not skill:isSingle() then
+            target:delStatusEffect(xi.effect.THIRD_EYE)
+        end
 
-    --handle Third Eye using shadowbehav as a guide
-    if attackType == xi.attackType.PHYSICAL and utils.thirdeye(target) then
-        skill:setMsg(xi.msg.basic.ANTICIPATE)
-        return 0
+        --handle Third Eye using shadowbehav as a guide
```

Comment:
`--` -> `-- `

surprised a ci rule didn't trip on that

Reply:
I'm more surprised at me not seing it

Reply:
there is another one like it in a section of code you didn't touch. in ability.lua

Reply:
Fixed all of the ones i could find